etc/profile-m-z/ssh-agent.profile

```diff
@@ -1,37 +1,62 @@
 # Firejail profile for ssh-agent
+# Description: OpenSSH authentication agent
 # This file is overwritten after every install/update
 quiet
 # Persistent local customizations
 include ssh-agent.local
 # Persistent global definitions
 include globals.local
 
+blacklist ${RUNUSER}/wayland-*
+
 # Allow ssh (blacklisted by disable-common.inc)
 include allow-ssh.inc
 
-blacklist /tmp/.X11-unix
-blacklist ${RUNUSER}/wayland-*
+ignore noexec ${RUNUSER}
 
 include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
 include disable-programs.inc
+include disable-shell.inc
+include disable-X11.inc
+include disable-xdg.inc
 
+include whitelist-common.inc
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
```

Comment:
So it does not need to access `$XDG_RUNTIME_DIR/keyring/ssh` or other ssh sockets in there?

Reply:
Not according to my observations no. Other parts of OpenSSH need that kind of access, like ssh, ssh-add, ssh-copy-id, ssh-key{gen,scan}... The authentication agent itself is kept 'dumb' on purpose (which is a good thing security-wise). We don't have profiles (yet) for ssh-add and the others mentioned.